holoviews/ipython/__init__.py

```diff
@@ -179,18 +179,21 @@ def __call__(self, *args, **params):
                                                                               fig='svg')
                         holoviews.archive.exporters = [svg_exporter] +\
                                                       holoviews.archive.exporters
-                OutputMagic.allowed['backend'] = list_backends()
-                OutputMagic.allowed['fig'] = list_formats('fig', backend)
-                OutputMagic.allowed['holomap'] = list_formats('holomap', backend)
+
+                Store.output_options = OutputOptions()
```

Comment:
Don't quite get this, ``Store.output_options`` is an instance but all the actual configuration is set on the class? Can it all be class based?